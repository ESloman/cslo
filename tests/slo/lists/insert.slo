var x = [1, 2, 3];
println(x);

x.insert(0, 99);  # should insert at the start: [99, 1, 2, 3]
println(x);

x.insert(2, 99);  # should insert at position 3 [99, 1, 99, 2, 3]
x.insert(100, 99);  # should insert at the end  [99, 1, 99, 2, 3, 99]
println(x);
