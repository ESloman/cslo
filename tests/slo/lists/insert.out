list[3]: [1, 2, 3]
list[4]: [99, 1, 2, 3]
list[6]: [99, 1, 99, 2, 3, 99]
